src/pages/AssetMapPage/AMDrawer.tsx

```diff
@@ -246,6 +247,10 @@ export const AMDrawer = () => {
             isSelect
             InputProps={{
               endAdornment: <Search color="neutral" />,
+              style: {
+                border: 'none',
+                paddingRight: '12px',
```

Comment:
Magnifying glass was too much to the right and this `Autocomplete` should not have a border ([design](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=2825%3A1498))